packages/core/src/modules/oob/OutOfBandModule.ts

```diff
@@ -563,13 +567,12 @@ export class OutOfBandModule {
 
     // TODO: for each did we should look for a connection with the invitation did OR a connection with theirDid that matches the service did
     for (const didOrService of services) {
-      // We need to check if the service is an instance of string because of limitations from class-validator
-      if (typeof didOrService === 'string' || didOrService instanceof String) {
-        // TODO await this.connectionsModule.findByTheirDid()
-        throw new AriesFrameworkError('Dids are not currently supported in out-of-band invitation services attribute.')
+      let did: string
+      if (typeof didOrService === 'string') {
+        did = didOrService
+      } else {
+        did = outOfBandServiceToNumAlgo2Did(didOrService)
       }
```

Comment:
We could also call invitationDids for this: 

```ts
const invitationDids = outOfBandInvitation.invitationDids

for (const invitationDid of invitationDids) {
....
}
```

Otherwise this may look a bit cleaner (but the 1st approach should work fine):

```suggestion
      const did = typeof didOrService === 'string' ? didOrService : outOfBandServiceToNumAlgo2Did(didOrService)
```